### ISSUE TITLE
e is undefined in timeout function

### ISSUE THREAD

**Issue by KamalAman** [fiixlabs/fiix-cmms-client-js#1]
in the index.js, e is undefined in the getTimeoutMS `createError_Timeout(e);`
```js

        if (getTimeoutMs()) {
            req.setTimeout(getTimeoutMs(), function () {
                try {
                    stream.abort()
                } catch (e) {

                }

                var response = {};

                response.error = createError_Timeout(e);

                callback(response);
            });
        }
```

If you use eslint, it will automatically catch errors like this so this wont happen.

**Comment by jitenbhardwaj** [comment]
@KamalAman Thank you. We took a note of your feedback and tracking it under our internal issue tracking system.

**Comment by jitenbhardwaj** [comment]
Fixed in version 2.8.2
